src/components/FormSectionHeader/FormSectionHeader.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Grid, makeStyles } from '@material-ui/core';
+
+const borderColor = '#404040';
+const useStyles = makeStyles({
+  titleBox: {
+    fontFamily: 'Avenir Next',
+    backgroundColor: borderColor,
+    color: 'white',
+    border: `3px solid ${borderColor}`,
+    letterSpacing: 1,
+    display: 'inline-block'
+  },
+  line: {
+    display: 'inline-block',
+    border: `1px solid ${borderColor}`,
+    width: '100%'
+  }
+});
+
+function FormSectionHeader({ title }) {
+  const classes = useStyles();
```

Comment:
@radamson would probably say we should use `styles` instead of `classes` here. ;)

Reply:
Appeased him